### PR TITLE
Add batch_size, batch_buffer, and batch_workers attributes for datastore

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
- Adds attributes `batch_size`, `batch_buffer`, and `batch_workers` attributes for PostgresConfig spec to examples in datastore API and reference doc.
- Adds attribute description tables in the datastore reference doc.

## Motivation and Context
Support 6.1.0 release

### CODE REVIEW

**Comment by hillaryfraley** [comment]
@echlebek a couple questions on this one:
- The default buffer_size for my PostgresConfig was `0` and not `1` (although I don't have a postgres config for real b/c I haven't ever set it up).
- Should this be added to the changelog/release notes?

**Comment by echlebek** [comment]
@hillaryfraley 

1. If `buffer_size` is not set, then the default will be `10000`.
2. This should be mentioned in the release notes, but most users will not need to set it.

**Comment by hillaryfraley** [comment]
Gah I should have said batch_size, not buffer. I'm sorry. I'll get this updated and resubmitted for review. Thank you.

File: content/sensu-go/6.1/operations/deploy-sensu/datastore.md [hunk 1]
```diff
@@ -166,12 +175,36 @@ example      | {{< code shell >}}created_by: admin{{< /code >}}
...
+type         | Integer
+example      | {{< code shell >}}batch_size: 0{{< /code >}}
+
 dsn          |      |
 -------------|------
 description  | Data source names. Specified as a URL or [PostgreSQL connection string][15]. The Sensu backend uses the golang pq library, which supports a [subset of the PostgreSQL libpq connection string parameters][4].
 required     | true
 type         | String
-example      | {{< code shell >}}dsn: "postgresql://user:secret@host:port/dbname"{{< /code >}}
+example      | {{< code shell >}}batch_workers: 0{{< /code >}}
```
Comment (palourde) [inline content/sensu-go/6.1/operations/deploy-sensu/datastore.md]: Looks like this line should moved to line 200 and the dsn example should be left there

**Review by palourde — CHANGES_REQUESTED** [review]

File: content/sensu-go/6.1/operations/deploy-sensu/datastore.md [hunk 2]
```diff
@@ -166,12 +175,36 @@ example      | {{< code shell >}}created_by: admin{{< /code >}}
...
+type         | Integer
+example      | {{< code shell >}}batch_size: 1{{< /code >}}
+
+batch_workers |      |
+-------------|------
+description  | Number of Goroutines sending data to PostgreSQL, as specified in the PostgreSQL configuration.<br>We recommend specifying a small multiple of the pool size, between 20 and 100, to optimize PostgreSQL latency and throughput.
+required     | true
+default      | Current PostgreSQL pool size
+type         | Integer
+example      | {{< code shell >}}batch_size: 0{{< /code >}}
```
Comment (hillaryfraley) [inline content/sensu-go/6.1/operations/deploy-sensu/datastore.md]:
```suggestion
example      | {{< code shell >}}batch_workers: 0{{< /code >}}
```

File: content/sensu-go/6.1/operations/deploy-sensu/datastore.md (hunk 1 above)
Comment (hillaryfraley) [reply content/sensu-go/6.1/operations/deploy-sensu/datastore.md]: yes, you're correct, thank you.

Comment (hillaryfraley) [inline content/sensu-go/6.1/operations/deploy-sensu/datastore.md]:
```suggestion
example      | {{< code shell >}}dsn: "postgresql://user:secret@host:port/dbname"{{< /code >}}
```

**Review by palourde — APPROVED** [review]
👍

**Pull request merged**